test/e2e/apps/daemon_set.go

```diff
@@ -89,7 +89,7 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 			e2elog.Logf("unable to dump pods: %v", err)
 		}
 		err = clearDaemonSetNodeLabels(f.ClientSet)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		framework.ExpectNoError(err)
```

Comment:
This has been fixed already as https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/daemon_set.go#L92
The following is also the same.

Maybe your local repository is not updated for the latest master. Could you check it?

Reply:
Sorry, I'll close this PR.